src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -52,11 +54,12 @@ public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs
 
         this.addressBook = new AddressBook(addressBook);
         this.userPrefs = new UserPrefs(userPrefs);
+        this.allPersons = this.addressBook.getPersonList();
         this.filteredPersons = new FilteredList<>(this.addressBook.getPersonList());
         this.filteredSales = new FilteredList<>(this.addressBook.getSaleList());
         this.filteredSales.setPredicate(x -> false);
         this.sortedPersons = new SortedList<>(this.filteredPersons);
-        this.updateSortedPersonList(DEFAULT_PERSON_COMPARATOR);
```

Comment:
this is a feature, to have the contact list sorted by default. is there a reason it is removed?

Reply:
Might have accidentally removed it when merging. Will fix it asap!